nuxeo-compound-documents-web/ftest/features/login-logout.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Login / Logout
```

Comment:
This feature file will be replace by the feature file(s) that we will add in the future, right?

Reply:
I added this for testing purposes, it is just copied from Web UI. The goal is to replace it with actual feature files in the future (I could even delete it before merging the PR).